### DISCUSSION TITLE
.cklb file ingestion

### DISCUSSION THREAD

**Discussion by ScanningAndPOAMing** [Vulnerator/Vulnerator#229] [Ideas]
It appears DISA is releasing a new version of STIG Viewer (STIG Viewer 3.x Beta). In the User Guide, it appears this new version creates .cklb checklist files vice the old .ckl files. Any chance vulnerator will be be updated to accept this new file format? 

Thanks.

**Comment by RBremmer** [comment]
Just verified the .cklb STIG checklist files generated by STIG Viewer 3 are NOT recognized by Vulnerator.  Is a fix in the works?

**Reply by CyberSecDef** [reply]
I can’t speak to the status of Vulnerator but I was able to duplicate cklb (and Scap, acas, ckl) functionality on the report generator on the cyber Trackr. Feel free to check it out. 

https://cyber.trackr.live

**Comment by Cajuncrook** [comment]
I am a avid user of Vulnerator and as others was forced to STIG Viewer 3.x and found myself unable to ingest .cklb files into Vulnerator, hoping vendor will provide a updated version and if so how soon will that be if ever. Vulnerator is a excellent tool that save me a ton of work.
